st2common/st2common/operators.py

```diff
@@ -161,6 +169,7 @@ def nexists(value, criteria_pattern):
     return value is None
 
 # operator match strings
+MATCH_GLOB = 'matchglob'
```

Comment:
As the description says - it's simpler. For simple glob matching, regexes are can be an overkill.